### PR TITLE
[BCMSAI] Update BCM SAI debian package to 4.2.1.3 (6.5.19 hsdk)

### PR DESCRIPTION
**Pull request by smaheshm**
**- Why I did it**

Update BCM SAI debian package to 4.2.1.3 (6.5.19 hsdk)

**- How I did it**

Build debain package with BCMSAI 4.2.1.3 and 6.5.19 hsdk.

**- How to verify it**

Manual tests
```
 admin@str-s6000-acs-8:~$ show version

SONiC Software Version: SONiC.bcmsai4.2-nogpl.0-dirty-20201001.191202
Distribution: Debian 10.6
Kernel: 4.19.0-9-2-amd64
Build commit: 79f21728
Build date: Thu Oct  1 19:45:40 UTC 2020
Built by: samaddik@samaddik-vm-01

Platform: x86_64-dell_s6000_s1220-r0
HwSKU: Force10-S6000
ASIC: broadcom
Serial Number: 1QBRX42
Uptime: 18:30:19 up  2:03,  1 user,  load average: 1.05, 0.92, 0.82

Docker images:
REPOSITORY                    TAG                                       IMAGE ID            SIZE
docker-teamd                  bcmsai4.2-nogpl.0-dirty-20201001.191202   ea43f1f2af6d        391MB
docker-teamd                  latest                                    ea43f1f2af6d        391MB
docker-sflow                  bcmsai4.2-nogpl.0-dirty-20201001.191202   8f7610eeb99a        395MB
docker-sflow                  latest                                    8f7610eeb99a        395MB
docker-orchagent              bcmsai4.2-nogpl.0-dirty-20201001.191202   fe9b0db5ff0e        405MB
docker-orchagent              latest                                    fe9b0db5ff0e        405MB
docker-nat                    bcmsai4.2-nogpl.0-dirty-20201001.191202   4286e9655aff        394MB
docker-nat                    latest                                    4286e9655aff        394MB
docker-fpm-frr                bcmsai4.2-nogpl.0-dirty-20201001.191202   49b027a3aed7        407MB
docker-fpm-frr                latest                                    49b027a3aed7        407MB
docker-snmp                   bcmsai4.2-nogpl.0-dirty-20201001.191202   1c8494bcbc8a        399MB
docker-snmp                   latest                                    1c8494bcbc8a        399MB
docker-router-advertiser      bcmsai4.2-nogpl.0-dirty-20201001.191202   4ca6adf3a696        359MB
docker-router-advertiser      latest                                    4ca6adf3a696        359MB
docker-platform-monitor       bcmsai4.2-nogpl.0-dirty-20201001.191202   95c21f4cfb46        433MB
docker-platform-monitor       latest                                    95c21f4cfb46        433MB
docker-lldp                   bcmsai4.2-nogpl.0-dirty-20201001.191202   cffe7dd2bacb        387MB
docker-lldp                   latest                                    cffe7dd2bacb        387MB
docker-database               bcmsai4.2-nogpl.0-dirty-20201001.191202   e0909d0e71b1        359MB
docker-database               latest                                    e0909d0e71b1        359MB
docker-sonic-telemetry        bcmsai4.2-nogpl.0-dirty-20201001.191202   a8518ff11a90        429MB
docker-sonic-telemetry        latest                                    a8518ff11a90        429MB
docker-dhcp-relay             bcmsai4.2-nogpl.0-dirty-20201001.191202   e1a2dbec8a2a        366MB
docker-dhcp-relay             latest                                    e1a2dbec8a2a        366MB
docker-sonic-mgmt-framework   bcmsai4.2-nogpl.0-dirty-20201001.191202   d379c1e24601        486MB
docker-sonic-mgmt-framework   latest                                    d379c1e24601        486MB
docker-syncd-brcm             bcmsai4.2-nogpl.0-dirty-20201001.191202   973d876f1731        542MB
docker-syncd-brcm             latest                                    973d876f1731        542MB



admin@str-s6000-acs-8:~$ show ip bgp summary

IPv4 Unicast Summary:
BGP router identifier 10.1.0.32, local AS number 65100 vrf-id 0
BGP table version 25
RIB entries 49, using 9016 bytes of memory
Peers 24, using 502080 KiB of memory
Peer groups 4, using 256 bytes of memory


Neighbhor      V     AS    MsgRcvd    MsgSent    TblVer    InQ    OutQ  Up/Down      State/PfxRcd  NeighborName
-----------  ---  -----  ---------  ---------  --------  -----  ------  ---------  --------------  --------------
10.0.0.1       4  65200       2177       2194         0      0       0  01:48:38                1  ARISTA01T2
10.0.0.5       4  65200       2184       2203         0      0       0  01:49:00                1  ARISTA03T2
10.0.0.9       4  65200       2200       2223         0      0       0  01:49:47                1  ARISTA05T2
10.0.0.13      4  65200       2196       2218         0      0       0  01:49:35                1  ARISTA07T2
10.0.0.17      4  65200       2193       2213         0      0       0  01:49:25                1  ARISTA09T2
10.0.0.21      4  65200       2201       2227         0      0       0  01:49:49                1  ARISTA11T2
10.0.0.25      4  65200       2194       2215         0      0       0  01:49:28                1  ARISTA13T2
10.0.0.29      4  65200       2182       2200         0      0       0  01:48:54                1  ARISTA15T2
10.0.0.33      4  64001       2189       2208         0      0       0  01:49:14                1  ARISTA01T0
10.0.0.35      4  64002       2180       2197         0      0       0  01:48:46                1  ARISTA02T0
10.0.0.37      4  64003       2193       2212         0      0       0  01:49:23                1  ARISTA03T0
10.0.0.39      4  64004       2184       2203         0      0       0  01:48:58                1  ARISTA04T0
10.0.0.41      4  64005       2197       2219         0      0       0  01:49:38                1  ARISTA05T0
10.0.0.43      4  64006       2192       2211         0      0       0  01:49:21                1  ARISTA06T0
10.0.0.45      4  64007       2195       2216         0      0       0  01:49:33                1  ARISTA07T0
10.0.0.47      4  64008       2199       2221         0      0       0  01:49:42                1  ARISTA08T0
10.0.0.49      4  64009       2194       2216         0      0       0  01:49:30                1  ARISTA09T0
10.0.0.51      4  64010       2181       2198         0      0       0  01:48:50                1  ARISTA10T0
10.0.0.53      4  64011       2178       2195         0      0       0  01:48:40                1  ARISTA11T0
10.0.0.55      4  64012       2187       2206         0      0       0  01:49:07                1  ARISTA12T0
10.0.0.57      4  64013       2202       2225         0      0       0  01:49:52                1  ARISTA13T0
10.0.0.59      4  64014       2177       2193         0      0       0  01:48:37                1  ARISTA14T0
10.0.0.61      4  64015       2202       2225         0      0       0  01:49:52                1  ARISTA15T0
10.0.0.63      4  64016       2186       2205         0      0       0  01:49:03                1  ARISTA16T0

Total number of neighbors 24

admin@str-s6000-acs-8:~$ show lldp neighbors | grep SysName
    SysName:      str-22ex-235ab08
    SysName:      ARISTA07T2
    SysName:      ARISTA07T2
    SysName:      ARISTA09T2
    SysName:      ARISTA09T2
    SysName:      ARISTA11T2
    SysName:      ARISTA11T2
    SysName:      ARISTA01T2
    SysName:      ARISTA01T2
    SysName:      ARISTA03T2
    SysName:      ARISTA03T2
    SysName:      ARISTA05T2
    SysName:      ARISTA05T2
    SysName:      ARISTA13T2
    SysName:      ARISTA13T2
    SysName:      ARISTA15T2
    SysName:      ARISTA15T2
    SysName:      ARISTA01T0
    SysName:      ARISTA02T0
    SysName:      ARISTA04T0
    SysName:      ARISTA03T0
    SysName:      ARISTA12T0
    SysName:      ARISTA11T0
    SysName:      ARISTA13T0
    SysName:      ARISTA14T0
    SysName:      ARISTA15T0
    SysName:      ARISTA16T0
    SysName:      ARISTA05T0
    SysName:      ARISTA06T0
    SysName:      ARISTA08T0
    SysName:      ARISTA07T0
    SysName:      ARISTA09T0
    SysName:      ARISTA10T0


    admin@str-s6000-acs-8:~$ show ip route summary
    Route Source         Routes               FIB  (vrf default)
    connected            26                   26                   
    static               1                    1                    
    ebgp                 24                   24                   
    ibgp                 0                    0                    
    ------
    Totals               51                   51                   

    admin@str-s6000-acs-8:~$ show arp
    Address     MacAddress         Iface            Vlan
    ----------  -----------------  ---------------  ------
    10.0.0.1    52:54:00:01:8f:a9  PortChannel0002  -
    10.0.0.5    52:54:00:20:53:8d  PortChannel0005  -
    10.0.0.9    52:54:00:5a:3c:56  PortChannel0008  -
    10.0.0.13   52:54:00:30:b3:41  PortChannel0011  -
    10.0.0.17   52:54:00:43:da:8a  PortChannel0014  -
    10.0.0.21   52:54:00:f4:12:cd  PortChannel0017  -
    10.0.0.25   52:54:00:f8:7c:e8  PortChannel0020  -
    10.0.0.29   52:54:00:ae:70:50  PortChannel0023  -
    10.0.0.33   52:54:00:36:4d:c7  Ethernet64       -
    10.0.0.35   52:54:00:25:ef:03  Ethernet68       -
    10.0.0.37   52:54:00:07:dc:96  Ethernet72       -
    10.0.0.39   52:54:00:d8:dd:c8  Ethernet76       -
    10.0.0.41   52:54:00:5b:f2:80  Ethernet80       -
    10.0.0.43   52:54:00:fd:84:b6  Ethernet84       -
    10.0.0.45   52:54:00:bb:00:36  Ethernet88       -
    10.0.0.47   52:54:00:b3:88:1f  Ethernet92       -
    10.0.0.49   52:54:00:0b:92:4e  Ethernet96       -
    10.0.0.51   52:54:00:29:d0:cd  Ethernet100      -
    10.0.0.53   52:54:00:2a:f2:32  Ethernet104      -
    10.0.0.55   52:54:00:85:c1:aa  Ethernet108      -
    10.0.0.57   52:54:00:62:db:2a  Ethernet112      -
    10.0.0.59   52:54:00:68:98:7b  Ethernet116      -
    10.0.0.61   52:54:00:73:12:13  Ethernet120      -
    10.0.0.63   52:54:00:77:df:9e  Ethernet124      -
    10.3.146.1  00:00:5e:00:01:64  eth0             -
    10.3.146.2  f4:b5:2f:79:b3:f0  eth0             -
    Total number of entries 26 
    admin@str-s6000-acs-8:~$ ping 10.0.0.63
    PING 10.0.0.63 (10.0.0.63) 56(84) bytes of data.
    64 bytes from 10.0.0.63: icmp_seq=1 ttl=64 time=14.1 ms
    ^C
    --- 10.0.0.63 ping statistics ---
    1 packets transmitted, 1 received, 0% packet loss, time 0ms
    rtt min/avg/max/mdev = 14.129/14.129/14.129/0.000 ms

```

**- Which release branch to backport (provide reason below if selected)**

- [ ] 201811
- [ ] 201911
- [ ] 202006

**- Description for the changelog**

BRCM_SAI_VER "4.2.1.3"
OCP_SAI_VER  "1.6.5"
BCM SAI changelog (git log --pretty=oneline REL_GA1_3.7.5.1...REL_GA_4.2.1.3)
```
aea987188f7d4942fceef4b0abbe2afbce7f791d Add kbp file
6f3e31a5b05d59ddfce6ee01eceb38f059789d3b Remove old jink files.
3f53c17415b946e67cf253417a8739894d0ba489 4.2 GA release
06dd199f993ec6bdf902d3ae3cdce87710f4d546 Remove old junk files.
fe47737987532b1397cef91ef5160c24d3137e6c Update brcm_sai_version.h
07d928935f28476a4890e498c8541d907b33f6a3 Update README.md
0d35f6ddb3b5915b3cf1e6a75a32656eff22d956 SAI 4.2 EA1 release.
f9012cd44772d7ce4667d95bb95932f1720b4de6 Add missing build directories.
88399ae16a2c39996ee514f9c7d6064d77de8960 Temporary doc update.
0076ce9ccf0cea296fd9fa2fc0828e6da82bc7f0 Update README.md
ba56abdb8127569a47d219a15e2c519436509aa8 Cleanup unwanted file.
93fd029fa1d5c1d1025be129e83b53979ef4f617 SAI 4.2 Engg drop
1475c241e25c91fc8cb93d7c14cd70194bad1328 CSP10503275 request to pull KB0029314 into 3.7

admin@str-s6000-acs-8:~$ bcmcmd bsv
bsv
BRCM SAI ver: [4.2.1.3], OCP SAI ver: [1.6.5], SDK ver: [sdk-6.5.19]
drivshell>
admin@str-s6000-acs-8:~$ 


```
**- A picture of a cute animal (not mandatory but encouraged)**

### CODE REVIEW

**Comment by smaheshm** [comment]
Earlier draft #5399

**Comment by smaheshm** [comment]
retest broadcom please

File: platform/broadcom/saibcm-modules/systems/linux/kernel/modules/Makefile [hunk 1]
```diff
@@ -24,7 +24,7 @@ LOCALDIR = systems/linux/kernel/modules
 
 include ${SDK}/make/Make.config
 
-subdirs=shared uk-proxy bcm-diag-full bcm-core bcm-net bcm-diag
+subdirs=
```
Comment (lguohan) [inline platform/broadcom/saibcm-modules/systems/linux/kernel/modules/Makefile:27]: why removed?

Comment (smaheshm) [reply platform/broadcom/saibcm-modules/systems/linux/kernel/modules/Makefile:27]: the "systems" folder is copied as is from "sdk-gpl-6.5.19" source. 

There were some changes added back to the files copied. If this change has to be there then I can add it back.

**Review by prsunny — APPROVED** [review]
lgtm

**Comment by lguohan** [comment]
can you package this into the opennsl-module? it is built but not packaged in the deb file.

linux-bcm-ptp-clock.ko

**Review by lguohan — APPROVED** [review]

**Comment by smaheshm** [comment]
retest vsimage please

**Comment by smaheshm** [comment]
rebuild vsimage please

**Pull request merged**